File: logica-programacao/python/035.py
# Receba um ano.
# Considere bissexto se:
# for divisível por 4
# e não for divisível por 100
# ou
# for divisível por 400
# Retorne True ou False.

ano = input("Ano: ")
int_ano = int(ano)

bissexto = (ano % 4 == 0 and ano % 100 != 0) or (ano % 400 == 0)

string_bissexto = f"O ano é bissexto?: {bissexto}"

print(string_bissexto)

File: logica-programacao/javascript/035.js
/* Receba um ano.
Considere bissexto se:

for divisível por 4
e não for divisível por 100
ou

for divisível por 400

Retorne True ou False. */

let ano = prompt("Ano: ");
let intAno = parseInt(ano);

let bissexto = (intAno % 4 === 0 && intAno % 100 !== 0) || (intAno % 400 === 0);

let stringBissexto = `O ano é bissexto? ${bissexto}`;
console.log(stringBissexto);